esp/src/src-react/components/controls/Chip.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from "react";
+import { styled } from '@fluentui/utilities';
+import { lighten, fade } from "@material-ui/core/styles";
+
+import { IStyleFunctionOrObject, classNamesFunction } from '@fluentui/react';
+import { IStyle, ITheme } from '@fluentui/style-utilities';
+
+import { StatusErrorFullIcon } from "@fluentui/react-icons-mdl2";
+
+interface IChipProps extends React.RefAttributes<HTMLDivElement> {
+    label: string;
+    color?: string;
+    onDelete?: Function;
+    className?: string;
+    theme?: ITheme;
+    styles?: IStyleFunctionOrObject<IChipStyleProps, IChipStyles>;
+}
+
+interface IChipStyleProps {
+    className?: string;
+    theme: ITheme;
+    color: string,
+}
+
+interface IChipStyles {
+    root: IStyle;
+    chipLabel: IStyle;
+    chipCloseIcon: IStyle;
```

Comment:
Is IStyle generic or are there more specific styles for different html types (this is a question for me to lookup)?

Reply:
It does seem to just be a generic key value pair of any types